web/sentry.client.config.js

```diff
@@ -0,0 +1,17 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  dsn: SENTRY_DSN || 'https://873c9b3ff9ce483cb07e0498445a3d5b@o114654.ingest.sentry.io/6604345',
```

Comment:
Assuming the hardcoded DSN is a 'dev' DSN? I think we should still have this value in .env for local development as well.